app/Main.hs

```diff
@@ -461,7 +484,17 @@ verifyPackage db paths name = do
   echoT ("Verifying package " <> runPackageName name)
   dependencies <- map fst <$> getTransitiveDeps db [name]
   let srcGlobs = map (pathToTextUnsafe . (</> ("src" </> "**" </> "*.purs")) . dirFor) dependencies
-  procs "purs" ("compile" : srcGlobs) empty
+  runPurs srcGlobs
+  where
+    runPurs :: [Text] -> IO ()
```

Comment:
I made this change, @paf31. Not sure if that's what you had in mind.

Reply:
Thanks, this is closer, but what I meant was to move this function to the top level, and to combine it with `pursCmd`. Could you please try that?

Reply:
I’ll give it a shot